turbine_models/tools/interface_tools.py

```diff
@@ -0,0 +1,110 @@
+import numpy as np
+from turbine_models.parser import Turbines
+from turbine_models.tools.make_power_curve import extract_power_curve
+from turbine_models.supported_turbines import missing_information_turbines
+
+def get_pysam_turbine_specs(turbine_name):
+    """Load turbine data from turbine-models library to use with PySAM wind simulation.
+
+    Args:
+        turbine_name (str): name of turbine in turbine-models library
+
+    Raises:
+        ValueError: if turbine is missing data.
+
+    Returns:
+        dict: turbine model dictionary formatted for PySAM.
+    """
+    if turbine_name in missing_information_turbines:
+        raise ValueError(f"{missing_information_turbines} turbine does not have enough information to model in PySAM.")
+    t_lib = Turbines()
+    turbine_specs = t_lib.specs(turbine_name)
+
+    if isinstance(turbine_specs,dict):
```

Comment:
I would recommend converting this check to `if not ...`, then raising the `ValueError` at the end, then dedenting the rest of the function's code as it won't be under an `if` statement.